generator.rb

```diff
@@ -1,69 +1,56 @@
+# frozen_string_literal: true
+
 # TODO: Naming and organization clean-up
 require 'zlib'
 require 'zip'
-require 'package'
 require 'json'
 require 'crack'
 require 'fhir_models'
 require 'pry-nav'
 require 'SecureRandom'
-require_relative './IGExtractor'
+require_relative './IG'
 require_relative './TestScriptWorkflow'
 require_relative './TestScriptGenerator'
 
 class Generator
-  include IGExtractor
-
   MAPPING = {
     'read' => 'read',
     'create' => 'create',
     'delete' => 'delete',
     'search-type' => 'search'
-  }
+  }.freeze
 
-  def igs 
-    @igs ||= load_igs
-  end 
-
-  def interactions_map
-    @interactions_map ||= map_interactions
-  end 
+  def igs
+    @igs ||= Dir.glob("#{Dir.getwd}/igs/*").each_with_object({}) do |package, store|
```

Comment:
Ideally you want to pass in a path to the IG you want to generate tests for, rather than assume it will be a subdirectory of where this tool is installed. This would require upstream changes in `driver.rb` too.